src/microbe_stage/gui/ExtinctionBox.tscn

```diff
@@ -33,22 +33,20 @@ _data = {
 
 [node name="ExtinctionBox" type="Control" node_paths=PackedStringArray("extinctionMenu", "loadMenu", "continueText", "continueButton")]
 layout_mode = 3
-anchors_preset = 15
-anchor_right = 1.0
-anchor_bottom = 1.0
-grow_horizontal = 2
-grow_vertical = 2
+anchors_preset = 0
```

Comment:
Which preset is this? I think the "fullscreen" preset should be selected, I think / that makes the most sense to me. Is it 0, I can't remember of the top off my head?